packages/web-frontend/src/containers/OverlayDiv/components/LandingPage/index.js

```diff
@@ -45,11 +45,11 @@ const ViewProjectsButton = styled(Button)`
 `;
 
 export const LandingPage = ({ isUserLoggedIn, isViewingProjects }) => {
-  const [isProjectsPageVisible, setIsProjectsPageVisible] = React.useState(false);
+  const [isProjectsPageVisible, setIsProjectsPageVisible] = React.useState(isViewingProjects);
```

Comment:
Pretty simple bug, I hadn't consider that we need to control whether we're looking at the projects page via internal state.